.github/workflows/ci.yaml

```diff
@@ -81,6 +81,18 @@ jobs:
       - run: yarn build:internal-dependencies
       - run: ./packages/devops/scripts/ci/validateTypesAndDbSchemaInSync.sh
 
+  validate_lockfile:
```

Comment:
Yarn recommends to validate the lockfile is up to date explicitly in a separate validation CI step